tests/app/submitter/test_submitter.py

```diff
@@ -0,0 +1,77 @@
+from unittest import TestCase
+
+from mock import patch, Mock, call, MagicMock
+from pika.exceptions import AMQPError
+
+from app import settings
+from app.submitter.submitter import RabbitMQSubmitter
+
+
+class TestSubmitter(TestCase):
+
+    def test_send_message_fails(self):
+        # Given
+        submitter = RabbitMQSubmitter()
+
+        # When
+        published = submitter.send_message(message={}, queue='test_queue')
+
+        # Then
+        self.assertFalse(published, 'send_message should fail to publish message')
```

Comment:
Why should it fail? test doesn't explain why. so if this test fails i wouldn't know what was wrong